src/App.js

```diff
@@ -1,24 +1,18 @@
 import logo from './logo.svg';
 import './App.css';
+import Button from './components/Button/Button'
 
 function App() {
+ 
+
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    
+    <>
+    <Button button={"Contact"} currentColor={"dark"} />
+    
+    <Button button={"Something"} currentColor={"grey"} />
```

Comment:
String can be directly pass without curly brackets.

```suggestion
    <Button button="Something" currentColor="grey" />
```